src/test/java/io/appium/java_client/drivers/options/OptionsBuildingTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.drivers.options;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.appium.java_client.android.options.EspressoOptions;
+import io.appium.java_client.android.options.UiAutomator2Options;
+import io.appium.java_client.android.options.localization.AppLocale;
+import io.appium.java_client.android.options.server.EspressoBuildConfig;
+import io.appium.java_client.android.options.signing.KeystoreConfig;
+import io.appium.java_client.ios.options.XCUITestOptions;
+import io.appium.java_client.ios.options.other.CommandTimeouts;
+import io.appium.java_client.ios.options.simulator.Permissions;
+import io.appium.java_client.mac.options.Mac2Options;
+import io.appium.java_client.remote.AutomationName;
+import io.appium.java_client.windows.options.RunScript;
+import io.appium.java_client.windows.options.WindowsOptions;
+import org.junit.Test;
+import org.openqa.selenium.Platform;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Duration;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@SuppressWarnings("ConstantConditions")
+public class OptionsBuildingTest {
+    @Test
+    public void canBuildXcuiTestOptions() throws MalformedURLException {
+        XCUITestOptions options = new XCUITestOptions();
+        assertEquals(options.getPlatformName(), Platform.IOS);
```

Comment:
`asssertEquals` accepts `expected` as the first parameter and `actual` as the second parameter

Reply:
and it seems it's relevant to all assertions in this class

Reply:
fixed